### PR TITLE
This commit adds the mockup payment site.

### PR DESCRIPTION
**Pull request by Oscar-Rydh**
A payment has a 10% chance to fail.
This still needs to be redirected to a receipt site

### CODE REVIEW

**Comment by Oscar-Rydh** [comment]
Fixade merge conflict i online editorn.

File: public_html/payment.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <br>
+                <?php echo "Att betala: <b>", $_SESSION["total_price"], "</b> kr <t>"; ?>
+                <input type="button" value="Utför betalning" onclick="return createPayment()" > 
+            </form>
+        </body>
+        <script type='text/javascript'>
+        function createPayment() 
+        {
+        // Adding 10 percent failure rate on payment
+            var randomSuccess=Math.floor((Math.random() * 10));
```
Comment (VictorWinberg) [inline public_html/payment.php:32]: Haha nice! 😆

**Review by VictorWinberg — APPROVED** [review]
LGTM

File: public_html/payment.php (hunk 1 above)
Comment (Oscar-Rydh) [reply public_html/payment.php:32]: ;)

Comment (VictorWinberg) [reply public_html/payment.php:32]: Hmm, the creatPayment method doesn't work for me tho. It never switches back to '/store.php'.

**Pull request merged**